http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpWsmDao.scala

```diff
@@ -153,6 +156,78 @@ class HttpWsmDao[F[_]](httpClient: Client[F], config: HttpWsmDaoConfig)(implicit
       )(onError)
     } yield res
 
+  override def getLandingZoneResources(billingProfileId: String, userToken: Authorization)(implicit
```

Comment:
Good point that this code needs to be shared. Another pattern we've done is put things in companion objects, example: https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeServiceInterp.scala#L851

This seems _okay_ to put in `WsmDAO` though, since it's just calling a couple WSM APIs and parsing the responses.

Reply:
I'm ok with having it in WsmDao for now...Making the other 2 methods private sound good to me!